manifests/init.pp

```diff
@@ -37,7 +38,7 @@
   Hash $default_options,
   Optional[Hash] $options,
 
-  Boolean $iptables_docker_chain = simplib::lookup('simp_options::firewall', { 'default_value' => false }),
+  Boolean $iptables_docker_chain,
```

Comment:
This can be removed, along with the associated logic now.

Reply:
I left it in in case users aren't able to get the newest iptables module, like if they're stuck on 6.0.0 or 6.1.0

Reply:
You mean...if they do something that is invalid per the `metadata.json`?

They *need* that `iptables` update due to the bug fixes. Let's not release a new module that works around fixed bugs if we can help it.